packages/clay-core/src/vertical-nav/VerticalNav.tsx

```diff
@@ -274,9 +290,15 @@ function VerticalNav<T extends Record<string, any> | string>({
 			</VerticalNavContext.Provider>
 		</Nav>
 	);
+	useEffect(() => {
+		if (!ariaLabel) {
+			console.error(messages['error']);
+		}
```

Comment:
We have this pattern in other components, you can use the warning package for this.

https://github.com/liferay/clay/blob/master/packages/clay-icon/src/index.tsx#L33-L36

Reply:
Done!